cmd/cli/commands/configure.go

```diff
@@ -15,32 +16,21 @@ func newConfigureCmd() *cobra.Command {
 	var draftModel string
 	var numTokens int
 	var minAcceptanceRate float64
+	var hfOverrides string
+	var reasoningBudget int64
 
 	c := &cobra.Command{
-		Use:    "configure [--context-size=<n>] [--speculative-draft-model=<model>] MODEL [-- <runtime-flags...>]",
+		Use:    "configure [--context-size=<n>] [--speculative-draft-model=<model>] [--hf_overrides=<json>] [--reasoning-budget=<n>] MODEL",
```

Comment:
We should be careful about adding things like hf_overrides to our API, we are getting into the territory of inventing a DMR API again.

Reply:
Personally I would go 1. and open pull request to fix in vllm or open an issue in vllm of why we need these args (when a model is integrated well into vLLM or llama.cpp, it typically needs no args):

https://github.com/vllm-project/vllm/issue

Also, in llama.cpp we can detect and set this based on model metadata, why can't we do that here?

Keep in mind the users have a solution, use llama.cpp .

I think 2. should be close to a last resort if there is great user demand for this specific thing on vLLM.

But if you feel strongly about it, keep it in.